HIRS_Utils/src/main/java/hirs/data/persist/tpm/PcrInfoShort.java

```diff
@@ -250,4 +296,18 @@ public final int hashCode() {
 
         return byteBuffer.array();
     }
+
+    private boolean isTpm1() {
+        // need to get an individual PCR and measure length to determine SHA1 v SHA 256
+        List<TPMMeasurementRecord> pcrs = this.getPcrComposite().getPcrValueList();
+        if (pcrs.size() == 0) {
+            // TODO figure out a reasonable case
```

Comment:
TODO. I think I may have actually written this line. I think what I suggest in the following line is reasonable.

Reply:
fixed